src/server/components/auth/controllers.ts

```diff
@@ -33,23 +29,25 @@ export function getLoginController(
   });
 }
 
-export async function postLoginController(
-  req: Request,
-  res: Response,
-  next: NextFunction
-): Promise<void> {
+export async function postLoginController(req: Request, res: Response, next: NextFunction): Promise<void> {
   const emailAddress = req.body.emailAddress?.trim();
 
-  try {
-    if (!isGovUKEmailAddress(emailAddress)) {
-      res.render("login", {
-        error: true,
-      });
-      return;
-    }
+  if (!isValidEmailAddress(emailAddress)) {
+    res.render("login", {
+      error: true,
+    });
+    return;
+  }
 
+  if (!isGovUKEmailAddress(emailAddress)) {
+    res.render("login", {
+      success: true,
+    });
+    logger.info("Non GOV.UK email entered");
```

Comment:
I reckon this is a warning. (Someone has attempted to log into list management, who has access to FCDO VPN, with a non govuk email)

```suggestion
    logger.warn(`${emailAddress} (non GOV.UK) attempted to log in`);
```